subprojects/packagefiles/theora/meson.build

```diff
@@ -0,0 +1,269 @@
+project(
+  'theora',
+  'c',
+  'cpp',
+  version: '1.1.1',
+  meson_version: '>= 0.59.0',
+  license: 'BSD-3-Clause',
+  default_options: ['c_std=c11'],
+)
+
+conf = configuration_data()
+
+enable_valgrind = get_option('valgrind-testing').allowed()
+
+test_env = []
+
+if enable_valgrind
+  valgrind_exe = find_program('valgrind')
+
+  conf.set('HAVE_VALGRIND', valgrind_exe.found())
```

Comment:
Since valgrind_exe discovered with `required: <default>`, it is mandatory and this code cannot ever be false -- the build would simply error out. Unless you use my above suggestion.